roles/cham/tasks/install.yml

```diff
@@ -0,0 +1,131 @@
+- name: Cham - Create streaming root directory
+  file: 
+    path: "{{ streaming_root }}"
+    state: directory
+    owner: root
+    group: root
+    mode: 0777
+
+- name: Cham - Create hls directory
+  file: 
+    path: "{{ streaming_root }}/{{ hls_dir }}"
+    state: directory
+    owner: root
+    group: root
+    mode: 0777
+
+- name: Cham - Create flv directory
+  file: 
+    path: "{{ streaming_root }}/{{ flv_dir }}"
+    state: directory
+    owner: root
+    group: root
+    mode: 0777
+
+- name: Cham - Create recordings directory
+  file: 
+    path: "{{ streaming_root }}/{{ recordings_dir }}"
+    state: directory
+    owner: root
+    group: root
+    mode: 0777
+
+- name: Cham - Link to public recordings directory
+  file:
+    src: "{{ streaming_root }}/{{ recordings_dir }}"
+    dest: "{{ doc_root }}/local_content/{{ recordings_dir }}"
+    state: link
+  when: public_recordings | bool 
+
+- name: Cham - Install nginx-rtmp-module
+  package:
+    name:
+      - libnginx-mod-rtmp
+    state: latest
+
+- name: Cham - Install ffmpeg to allow transcoding of videos
+  package:
+    name:
+      - ffmpeg
+    state: latest
+
+- name: Cham - nginx config - copy http context declarations
+  template:
+    src: cham.conf.j2
+    dest: "{{ nginx_config_dir }}/cham.conf"
+    owner: root
+    group: root
+    mode: 0755
+
+- name: Cham - nginx config - Copy nginx rtmp webserver configuration file (with VAAPI hardware encoding support)
+  template:
+    src: rtmp.vaapi.conf.j2
+    dest: "{{ nginx_config_dir }}/../rtmp.conf"
+    owner: root
+    group: root
+    mode: 0755
+
+- name: Cham - nginx config - Copy nginx rtmp webserver configuration file (without hardware encoding support)
+  template:
+    src: rtmp.conf.j2
+    dest: "{{ nginx_config_dir }}/../rtmp.conf"
+    owner: root
+    group: root
+    mode: 0755
+
+- name: Cham - nginx config - Enable nginx rtmp webserver configuration file
+  lineinfile: 
+    path: "{{ nginx_config_dir }}/../nginx.conf"
+    line: "include {{ nginx_config_dir }}/../rtmp.conf;"
```

Comment:
could define {{ nginx_dir }} in default_vars and use that in place of {{ nginx_config_dir }}../

Reply:
agree with that. can you please do that :)